indico/modules/events/logs/client/js/components/LogEntryModal.jsx

```diff
@@ -116,6 +136,10 @@ export default class LogEntryModal extends React.Component {
             >
               <Translate>Previous</Translate>
             </IButton>
+
+            <IButton onClick={this.relatedEntries} disabled={!this._hasRelatedEntries()}>
```

Comment:
Should we also hide/disable it if the current metadata query already matches this entry's metadata?

Reply:
I think that would be more confusing unless we have a good way to show in the UI the reason why the button is hidden/disabled. At the same time, I don't think it harms much to show the button even if it will trigger the same results. I would keep this as a nice to have.